### PR TITLE
chore: fix lib dispatch

### PR DESCRIPTION
**Pull request by appflowy**
### Feature Preview

---

#### PR Checklist

- [ ] My code adheres to [AppFlowy's Conventions](https://docs.appflowy.io/docs/documentation/software-contributions/conventions)
- [ ] I've listed at least one issue that this PR fixes in the description above.
- [ ] I've added a test(s) to validate changes in this PR, or this PR only contains semantic changes.
- [ ] All existing tests are passing.

### CODE REVIEW

File: frontend/rust-lib/dart-ffi/src/lib.rs [hunk 1]
```diff
@@ -120,57 +191,28 @@ pub extern "C" fn async_event(port: i64, input: *const u8, len: usize) {
...
+  DART_APPFLOWY_CORE.dispatch(request, port, None);
 }
 
 #[no_mangle]
-pub extern "C" fn sync_event(input: *const u8, len: usize) -> *const u8 {
-  let request: AFPluginRequest = FFIRequest::from_u8_pointer(input, len).into();
-  #[cfg(feature = "sync_verbose_log")]
-  trace!("[FFI]: {} Sync Event: {:?}", &request.id, &request.event,);
+pub extern "C" fn sync_event(_input: *const u8, _len: usize) -> *const u8 {
+  error!("unimplemented sync_event");
```
Comment (Horusiath) [inline frontend/rust-lib/dart-ffi/src/lib.rs:199]: Is this safe?

File: frontend/rust-lib/dart-ffi/src/lib.rs [hunk 2]
```diff
@@ -85,26 +130,52 @@ pub extern "C" fn init_sdk(_port: i64, data: *mut c_char) -> i64 {
...
-
-  // let isolate = allo_isolate::Isolate::new(port);
-  *APPFLOWY_CORE.0.lock().unwrap() = runtime.block_on(async move {
-    Some(AppFlowyCore::new(config, cloned_runtime, log_stream).await)
-    // isolate.post("".to_string());
+  let (sender, mut task_rx) = mpsc::unbounded_channel::<Task>();
+  let handle = std::thread::spawn(move || {
+    let local_set = LocalSet::new();
+    let runtime = Builder::new_current_thread().enable_all().build().unwrap();
+    runtime.block_on(async {
```
Comment (Horusiath) [inline frontend/rust-lib/dart-ffi/src/lib.rs:146]: With that I think we still execute everything in a single thread, but at least this time this thread is not UI one.

I have few ideas how we could to more parallel work here, but this can be separate PR.

File: frontend/rust-lib/dart-ffi/src/lib.rs (hunk 1 above)
Comment (appflowy) [reply frontend/rust-lib/dart-ffi/src/lib.rs:199]: yes. not use yet

File: frontend/rust-lib/dart-ffi/src/lib.rs (hunk 2 above)
Comment (appflowy) [reply frontend/rust-lib/dart-ffi/src/lib.rs:146]: Nice. FYI. I want to fix dispatch ASAP and then improve it later.
